contentcuration/contentcuration/frontend/channelEdit/views/sync/SyncResourcesModal.vue

```diff
@@ -8,15 +8,22 @@
       :submitDisabled="!continueAllowed"
       :submitText="$tr('continueButtonLabel')"
       :cancelText="$tr('cancelButtonLabel')"
+      :size="600"
```

Comment:
with reduced padding inside the modal, we can shrink down to `525`
```suggestion
      :size="525"
```